src/forms/FormControl.scss

```diff
@@ -1,27 +1,23 @@
-// stylelint-disable primer/typography, primer/borders, primer/spacing, primer/box-shadow, max-nesting-depth
+// stylelint-disable primer/typography, primer/borders, primer/spacing, selector-max-type, selector-max-specificity, selector-no-qualifying-type,  max-nesting-depth
 
-// group label, field, caption and error message
-.FormGroup {
+// groups label, field, caption and inline error message
+.FormControl {
```

Comment:
Would these class name changes cause a breaking change for folks using `.FormGroup`?

Reply:
This is all brand spankin' new CSS so we don't need to worry 😄 I'm the only consumer so far ✨